meetings/api/migrations/0001_create_account_for_osf.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-07-12 16:31
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from django.contrib.auth.management import create_permissions
+
+
+class Migration(migrations.Migration):
+
+    def create_user_for_osf(apps, schema_editor):
+        User = apps.get_model("auth", "User")
+        Permission = apps.get_model("auth", "Permission")
+        apps.models_module = True
+        create_permissions(apps, verbosity=0)
+        apps.models_module = None
+
+        osfPassword = 'a very secret password' # make api call to OSF to get password?
```

Comment:
Yeah, I am not sure. We need something to authenticate this osf user with. Is there any other information we could use to do this? @Ghalko